modules/ROOT/pages/hostname.adoc

```diff
@@ -0,0 +1,7 @@
+= Setting a Hostname
+
+To set a custom hostname for your system, write to the `/etc/hostname`
+file, which acts as machine-specific state.  This can be done
+via Anaconda, or with a privileged container.
```

Comment:
> or with a privileged container

do you mean in a `podman build` workflow and shipped as an OS update.. or imperatively on the command line?